modules/distr-tracing-tempo-config-query-rbac.adoc

```diff
@@ -0,0 +1,96 @@
+// Module included in the following assemblies:
+//
+// * observability/distr_tracing/distr_tracing_tempo/distr-tracing-tempo-configuring.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="distr-tracing-tempo-config-query-rbac_{context}"]
+= Query role based access control (RBAC)
+
+Query RBAC is a feature that allows users to see attributes only for spans that originate from namespaces that they have access to.
+Which means users will be able to see traces from all namespaces, however the attributes will be filtered based on the namespaces they have access to,
+except for the attributes `service.name` and `k8s.namespace.name` which are always kept.
+
+.Sample Tempo CR with enabled multitenancy and enabled query RBAC
+[source,yaml]
+----
+apiVersion: tempo.grafana.com/v1alpha1
+kind:  TempoStack
+metadata:
+  name: simplest
+  namespace: chainsaw-multitenancy
+spec:
+  storage:
+    secret:
+      name: minio
+      type: s3
+  storageSize: 1Gi
+  resources:
+    total:
+      limits:
+        memory: 2Gi
+        cpu: 2000m
+  tenants:
+    mode: openshift
+    authentication:
+      - tenantName: dev
+        tenantId: "1610b0c3-c509-4592-a256-a1871353dbfb"
+  template:
+    gateway:
+      enabled: true # <1>
+    rbac:
+      enabled: true # <2>
+    queryFrontend:
+      jaegerQuery:
+        enabled: false # <3>
+----
+
+<1> Must be set to `true`.
+<2> Must be set to `true`.
+<3> Must be set to `false`.
+
+:_mod-docs-content-type: REFERENCE
+[id="distr-tracing-tempo-config-query-rbac-configure_{context}"]
+== Configuration
+
+The following example gives OpenShift authenticated users access to the Tempo `dev` tenant and read attributes from the project `project1`.
+
+.ClusterRole and ClusterRoleBinding to give authenticated users access to the `dev` tenant.
+[source,yaml]
+----
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: tempo-dev-read
+rules:
+- apiGroups: [tempo.grafana.com]
+  resources: [dev]  # <1>
+  resourceNames: [traces]
+  verbs: [get]
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: tempo-dev-read
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: tempo-dev-read
+subjects:
+subjects:
+  - kind: Group
+    apiGroup: rbac.authorization.k8s.io
+    name: system:authenticated # <2>
+----
+
+<1> Refers to the tenant name  defined in the `TempoStack` custom resource.
+<2> Refers to all authenticated OpenShift users.
+
+The second step is to give authenticated users access to the project `project1`.
+
+.Command to give authenticated users access to the project `project1`.
+[source,bash]
+----
+oc adm policy add-role-to-group view system:authenticated -n tutorial-application-backend2  # <1>
```

Comment:
For the example, I think giving access to all authenticated users is too broad access, I'd suggest e.g.
```
oc adm policy add-role-to-user view alice -n tutorial-application-backend2
```

(using `alice` from the example here: https://docs.openshift.com/container-platform/4.18/authentication/using-rbac.html#adding-roles_using-rbac)